utils/config.ts

```diff
@@ -73,6 +73,31 @@ export const FeatureFlagType = t.union([
 ]);
 export type FeatureFlagType = t.TypeOf<typeof FeatureFlagType>;
 
+const isMap = (s: t.mixed): s is ReadonlyMap<string, Ulid> => s instanceof Map;
+
+export const UlidMapFromString = new t.Type<ReadonlyMap<string, Ulid>, string>(
+  "UlidMapFromString",
+  isMap,
+  (s, c) => {
+    try {
+      if (typeof s !== "string") {
+        return t.failure(s, c);
+      }
```

Comment:
Maybe we can extract this if statement from the try block cause it is not thrworing any exception